File: LOO 2°Parcial/Simulador de Vehiculos CSharp/Simulador de Vehiculos CSharp/VehiculoDetenidoException.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Simulador_de_Vehiculos_CSharp
{
    class VehiculoDetenidoException : Exception
    {
        public VehiculoDetenidoException() : base("\nEl vehiculo ya esta detenido.")
        {

        }
        public String toString()
        {
            return base.ToString();
        }
    }
}

File: LOO 2°Parcial/Simulador de Vehiculos Java/src/Clases/VehiculoDetenidoException.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */

package Clases;

/**
 * 
 * @author Dimitri Medina Novelo
 */
public class VehiculoDetenidoException extends Exception{
    
    public VehiculoDetenidoException(){
      super("\nEl vehiculo ya esta detenido.");
    }
    
    @Override
    public String toString(){
      return super.toString();
    }
}
